src/reducers/snapshot.ts

```diff
@@ -352,13 +352,13 @@ export default {
           pendingSave: { duosDataset: { $set: false } },
         }),
 
-      [LOCATION_CHANGE]: (state, action: any) => {
+      [LOCATION_CHANGE]: (state: SnapshotState, action: any) => {
         // Don't reset state if the only change was query parameters
         const path = action.payload?.location?.pathname;
         if (path === state.lastPath) {
           return immutable(state, {
-            lastPath: { $set: path },
-          });
+            lastPath: { $set: path as string },
```

Comment:
I'm surprised you have to do the casts here. Does typescript not type interpolate correctly for these?

Reply:
Yeah, weirdly there were a couple of places where the type couldn't be inferred and without making it explicit the server wouldn't run

Reply:
Here's the ticket! https://broadworkbench.atlassian.net/browse/DR-3177